src/Mod/TechDraw/App/DrawPage.cpp

```diff
@@ -238,7 +239,7 @@ int DrawPage::addView(App::DocumentObject *docObj)
         return -1;
 
     //position all new views in center of Page (exceptDVDimension)
-    DrawView* view = dynamic_cast<DrawView*>(docObj);
+    DrawView* view = static_cast<DrawView*>(docObj);

```

Comment:
Be careful. Here you use a static_cast which returns an invalid pointer when docObj is not of type 'DrawViewDimension'. Directly afterwards you check for the type which is OK but further down you again use "view" without checking its type before.

Here it's better to use a dynamic_cast and then check the returned pointer if it's null and further down, too.